MVP Module/Source/Feed Presentation/FeedLoadingErrorViewModel.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  FeedLoadingErrorViewModel.swift
+//  MVP
+//
+//  Created by Ksenia on 21.07.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+struct FeedLoadingErrorViewModel {
+	let error: String
```

Comment:
The `error` could be an optional `String?`, to better represent the information that the view can display. Instead of the `View` protocol defining two methods (`show` and `hide`), we can accomplish the same behavior displaying a `ViewModel` with an optional value.

For example, in the `didStartLoadingFeed()` method, the Presenter can call  `feedErrorView.display(FeedErrorViewModel(message: .none))` to hide the error.
```suggestion
	let error: String?
```